1.12.7 hotfixes (do not merge until planetme sees the balancing commits)

trainvoi opened the pull request.

trainvoi committed:
fix steel recycling recipe

bruberu reviewed (approved):
this does indeed fix #857

trainvoi committed:
add fine sand -> sand recipes

trainvoi committed:
titanium chain reblancing

this should fix the fact that titanium chain is both chlorine and magnesium negative

now it should be properly chlorine negative and magnesium neutral

trainvoi committed:
fix magnalum recycling recipe

og recipe was electrolyzer

trainvoi committed:
fix selenate formula

trainvoi committed:
remove TiCl4 changes, chanced output of Mg doubled.

trainvoi committed:
nerf ilmenite chlorination

trainvoi committed:
fix FeCl3 recipes

trainvoi committed:
fix FeCl3/AlCl3 stuff

trainvoi committed:
fix formula

trainvoi committed:
more formulas

trainvoi committed:
fix lang

trainvoi committed:
standardize calcium hydroxide recipes

trainvoi committed:
standardize calcium hydroxide recipes

trainvoi committed:
Merge branch '1.12.7-hotfixes' of https://github.com/trainvoi/Supersymmetry into 1.12.7-hotfixes

trainvoi committed:
Update ChemistryOverhaul.groovy

trainvoi committed:
fix nitrochlorobenzen stuff

trainvoi committed:
reduce nitration mixture recipe time from 25s to 8s

trainvoi committed:
Update ChemistryOverhaul.groovy

trainvoi committed:
fix FeCl3

trainvoi committed:
fix sand recipe

trainvoi committed:
add ferrocene recipe

trainvoi committed:
oil proc recipe time reblance

trainvoi committed:
blacklist breweries, raw/high-oct gasoline, steam/hydro-cracked light/heavy fuel

trainvoi committed:
more jei removal of hydro-cracked fluids

trainvoi committed:
removing fermenter

trainvoi committed:
unblacklist LCR

trainvoi committed:
black list industrial steam casing

trainvoi committed:
balance chlorinated CH3Cl mixture recipe

spacesailormarie commented:
gyatt

trainvoi committed:
somehow this recipes got deleted

found them in the old SUSY versions.

trainvoi committed:
Merge branch 'master-hotfixes' into 1.12.7-hotfixes

trainvoi committed:
remove toluene from nitroaniline recipe
